Formula/gupnp-tools.rb

```diff
@@ -3,6 +3,7 @@ class GupnpTools < Formula
   homepage "https://wiki.gnome.org/GUPnP/"
   url "https://download.gnome.org/sources/gupnp-tools/0.10/gupnp-tools-0.10.1.tar.xz"
   sha256 "4ea96d167462b3a548efc4fc4ea089fe518d7d29be349d1cce8982b9ffb53b4a"
+  license "GPL-2.0-or-later"
```

Comment:
```suggestion
  license all_of: ["GPL-2.0-or-later", "LGPL-2.0-or-later"]
```
https://metadata.ftp-master.debian.org/changelogs//main/g/gupnp-tools/gupnp-tools_0.10.0-2_copyright

Reply:
Good call!